tests/mutations/test_mutations.py

```diff
@@ -95,3 +97,245 @@ def test_delete_with_filters(mutation, fruits):
+    result = mutation(
+        "{ geofield: createGeoField(data: { multiPoint: "
+        + str(multi_point)
+        + " }) { id } }"
+    )
+    assert not result.errors
+    assert (
+        deep_tuple_to_list(
+            GeosFieldsModel.objects.get(
+                id=result.data["geofield"]["id"]
+            ).multi_point.tuple
+        )
+        == multi_point
+    )
+
+    # Test for multiLineString
+    multi_line_string = [
+        [[0.0, 0.0], [1.0, 1.0]],
+        [[1.0, 1.0], [-1.0, -1.0]],
+        [[2.0, 2.0], [-2.0, -2.0]],
+    ]
+    result = mutation(
+        "{ geofield: createGeoField(data: { multiLineString: "
+        + str(multi_line_string)
+        + " }) { id } }"
+    )
+    assert not result.errors
+    assert (
+        deep_tuple_to_list(
+            GeosFieldsModel.objects.get(
+                id=result.data["geofield"]["id"]
+            ).multi_line_string.tuple
+        )
+        == multi_line_string
+    )
+
+    # Test for multiPolygon
+    multi_polygon = [
+        [
+            [[-1.0, -1.0], [-1.0, 1.0], [1.0, 1.0], [1.0, -1.0], [-1.0, -1.0]],
+            [[-2.0, -2.0], [-2.0, 2.0], [2.0, 2.0], [2.0, -2.0], [-2.0, -2.0]],
+        ],
+        [
+            [[-1.0, -1.0], [-1.0, 1.0], [1.0, 1.0], [1.0, -1.0], [-1.0, -1.0]],
+            [[-2.0, -2.0], [-2.0, 2.0], [2.0, 2.0], [2.0, -2.0], [-2.0, -2.0]],
+        ],
+    ]
+    result = mutation(
+        "{ geofield: createGeoField(data: { multiPolygon: "
+        + str(multi_polygon)
+        + " }) { id } }"
+    )
+    assert not result.errors
+    assert (
+        deep_tuple_to_list(
+            GeosFieldsModel.objects.get(
+                id=result.data["geofield"]["id"]
+            ).multi_polygon.tuple
+        )
+        == multi_polygon
+    )
+
+
+@pytest.mark.skipif(
+    not settings.GEOS_IMPORTED,
+    reason="Test requires GEOS to be imported and properly configured",
+)
+@pytest.mark.django_db(transaction=True)
+def test_update_geo(mutation):
+    from tests.models import GeosFieldsModel
+
+    geofield_obj = GeosFieldsModel.objects.create()
+
+    assert geofield_obj.point is None
+    assert geofield_obj.line_string is None
+    assert geofield_obj.polygon is None
+    assert geofield_obj.multi_point is None
+    assert geofield_obj.multi_line_string is None
+    assert geofield_obj.multi_polygon is None
+
+    id = str(geofield_obj.id)
+
+    # Test for point
+    point = [0.0, 1.0]
+    result = mutation(
+        "{ geofield: updateGeoFields(data: { id: "
+        + id
+        + ", point: "
+        + str(point)
+        + " }) { id } }"
+    )
+    assert not result.errors
+    geofield_obj.refresh_from_db()
+    assert deep_tuple_to_list(geofield_obj.point.tuple) == point
+
+    # Test for lineString
+    line_string = [[0.0, 0.0], [1.0, 1.0]]
+    result = mutation(
+        "{ geofield: updateGeoFields(data: { id: "
+        + id
+        + ", lineString: "
+        + str(line_string)
+        + " }) { id } }"
```

Comment:
Ditto (and for all other cases =P)

Reply:
Done d6584a21713a259f4513278df8f0afb19b27131c